src/index.js

```diff
@@ -0,0 +1,237 @@
+      state.lon = response.data[0].lon;
+      console.log(state.lat);
+      console.log(state.lon);
+      getRealTemp();
+    })
+    .catch((error) => {
+      console.log('Error!', error);
+    });
+};
+
+const getRealTemp = () => {
+  console.log(state.lat);
+  console.log(state.lon);
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: state.lat,
+        lon: state.lon,
+      },
+    })
+    .then((response) => {
+      console.log(response);
+      // const tempValueK = response.data.current.temp;
+      const tempValueK = response.data.current.temp;
+      const tempValueF = ((tempValueK - 273.15) * 9) / 5 + 32;
+      const tempF = Math.floor(tempValueF);
+      state.tempF = tempF;
+      displayRealTemp(tempF);
+      landscapeChangeAction();
+      tempValueColorAction();
+    })
+    .catch((error) => {
+      console.log('Error!', error);
+    });
+};
+
+const displayRealTemp = (tempF) => {
+  const tempValueElement = document.getElementById('tempValue');
+  // tempValueElement.textContent = state.tempF;
+  tempValueElement.textContent = tempF;
+};
+
+// Function to increase temp Value by clicking up arrow
+const incTempAction = () => {
+  state.tempValue += 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
+};
+
+// Function to decrease temp Value by clicking up arrow
+const decTempAction = () => {
+  state.tempValue -= 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
+};
+
+// Function to change color of temp value according to temp range
+const tempValueColorAction = () => {
+  console.log('tempValueColor being called');
+  // document.getElementById('tempValue').style.color = 'orange';
+  // state.color = 'orange';
+  const tempColorElement = document.getElementById('tempValue');
+  // tempColorElement.style.color = 'orange';
+  // tempColorElement.className = color;
+  if (state.tempValue <= 49) {
+    tempColorElement.style.color = 'teal';
+  } else if (50 <= state.tempValue && state.tempValue <= 59) {
+    tempColorElement.style.color = 'green';
+  } else if (60 <= state.tempValue && state.tempValue <= 69) {
+    tempColorElement.style.color = 'yellow';
+  } else if (70 <= state.tempValue && state.tempValue <= 79) {
+    tempColorElement.style.color = 'orange';
+  } else {
+    tempColorElement.style.color = 'red';
+  }
+};
+
+// Function to change landscape of temp value according to temp range
+const landscapeChangeAction = () => {
+  const landscapeElement = document.getElementById('landscape');
+  landscapeElement.textContent = state.landscape;
+
+  if (state.tempValue <= 59) {
+    landscapeElement.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    state.background = "url('../assets/winter.jpg')";
+    document.querySelector('html').style.backgroundImage =
```

Comment:
Instead of accessing `document.querySelector('html').style.backgroundImage` in each if/elif block, you could create a variable on line 106:
```javascript
const htmlElement = document.querySelector('html').style.backgroundImage;
```

Then you could use `htmlElement` in each block instead of the long statement